### PR TITLE
refactor: slight optimization

### PR DESCRIPTION
**Pull request by MoskalykA**
optimization is explained by this article:  https://www.builder.io/blog/double-equal-vs-triple-equal

### CODE REVIEW

**Review by CsokiHUN — APPROVED** [review]
Looks good, thanks

**Review by CsokiHUN — APPROVED** [review]

File: [core]/esx_context/index.html [hunk 1]
```diff
@@ -390,8 +390,8 @@
       });
 
       window.addEventListener("keydown", (e) => {
-        if (e.key == "Escape" || e.key == "Backspace") {
-          if (e.target.tagName.toLowerCase() == "input") {
+        if (e.key === "Escape" || e.key === "Backspace") {
```
Comment (jrence) [inline [core]/esx_context/index.html:393]: may i know why ===  or miss type  ? for are they same???

Comment (MoskalykA) [reply [core]/esx_context/index.html:393]: > may i know why === or miss type ? for are they same???

I didn't understand your comment

Comment (Thekuca) [reply [core]/esx_context/index.html:393]: === is faster since == checks for the type of the variable and then compares while === checks if they are equal

**Review by jrence — APPROVED** [review]

**Pull request merged**